src/libcore/fmt/mod.rs

```diff
@@ -1576,10 +1576,10 @@ impl<'a> Formatter<'a> {
     /// impl fmt::Debug for Foo {
     ///     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
     ///         fmt.debug_struct("Foo")
-    ///             .field("bar", &self.bar)
-    ///             .field("baz", &self.baz)
-    ///             .field("addr", &format_args!("{}", self.addr))
-    ///             .finish()
```

Comment:
for what it's worth, this is how rustfmt formats it currently:

https://play.rust-lang.org/?gist=90c28f98c5dc3e5c98b1a4a16d7aee96&version=nightly

(try pressing the 'Format' button)

Reply:
Urg. It's not aligned with the first line. What's the point then? :-/